contents/handbook/engineering/developing-locally.md

```diff
@@ -310,6 +325,8 @@ You can also use [pyenv](https://github.com/pyenv/pyenv) if you wish to manage m
     ```bash
     uv venv env --python 3.11
     ```
+   
+   > **Friendly tip:** Creating an env could raise a `Failed to parse` warning related to `pyproject.toml`. However, you should still see the `Activate with:` line at the very end, which means that your env was created successfully.
```

Comment:
this one feels like a flox/pyproject bug/problem... @Twixes might care but it is a useful tip for now regardless :)

Reply:
I didn't test `flox` approach, only the manual one. The full error is name-related, but not sure if it's intentional or just my local issue (doesn't seem like it though).

```
`pyproject.toml` is using the `[project]` table, but the required `project.name` field is not set
```